test/test_ufbrw.F90

```diff
@@ -129,6 +134,7 @@ program test_ufbrw
           write(55,'(5(1x,f8.2))')arr(1:5,1:irt)
           call openmb(50,subset,idate)
           call ufbint(50,arr,10,irt,jrt,'POB QOB TOB UOB VOB')
+          call ufbint(50,arr,10,irt,jrt,'POB QOB TOB UOB VOB')
```

Comment:
Maybe I'm missing something, but why do we have two identical calls to ufbint() in succession, both with all of the same arguments?

Reply:
DRS stacking test